app/controllers/application_controller.rb

```diff
@@ -47,4 +48,8 @@ def set_host
         Rails.application.routes.default_url_options[:host] = request.host_with_port || Rails.application.config.slidehub.root_url
       end
     end
+
+    def clear_session
+      ActiveRecord::SessionStore::Session.where(["updated_at < ?", Devise.remember_for.ago]).delete_all
```

Comment:
__[RuboCop]__ <sub>posted by Sider</sub>

Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.